practice/diskspace.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+function check_diskspace {
+	local percent
+
+	percent=$(list_numbers)
+
+	if [ -n "${percent}" ]
+	then
+		percent=$(list_percent)
+
+		echo "In these Filesystems more than 80% storage space is full:"
```

Comment:
I suggest paying attention to the correctness of the display messages too. Make sure that the messages are grammatically correct.